persist/src/main/resources/db/migration/V3_9_AddSitePicture.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE SITES
+ADD COLUMN picture TYPE VARCHAR(500);
```

Comment:
We need separate urls for prod and development so we won't be able to have a dedicated default picture from S3. However if we want to store default urls in production we can add those after if we want to avoid null